Modules/LDAPObject.c

```diff
@@ -312,36 +308,34 @@ attrs_from_List(PyObject *attrlist, char ***attrsp)
 
         for (i = 0; i < len; i++) {
             attrs[i] = NULL;
-            item = PySequence_Fast_GET_ITEM(seq, i);
+            item = PySequence_GetItem(seq, i);
```

Comment:
This returns a new reference, item needs to be decref'd.

Reply:
https://github.com/python-ldap/python-ldap/compare/bad8aef9dc8c74049a73784c3bb23a1c9bbd0ce1..b45e0c3826c5c41b8e1326f1a9c46ff00bbe75df